docs/admin_api/user_admin_api.rst

```diff
@@ -119,10 +119,11 @@ from a previous call.
 The parameter ``limit`` is optional but is used for pagination, denoting the
 maximum number of items to return in this call. Defaults to ``100``.
 
-The parameter ``user_id`` is optional and can be used to filter by user id.
+The parameter ``user_id`` is optional and filters to only users with user IDs
+that contain this value. This parameter is ignored when using the ``name`` parameter.
 
-The parameter ``name`` is optional and can be used to list only users with the
-local part of the user ID or display name that contain this value.
+The parameter ``name`` is optional and filters to only users with user ID localparts
```

Comment:
"... filters to **show** only users..."?

Reply:
"... filters to only **return** users..." perhaps?

Reply:
Yes, makes sense.